meeting-notes/2018/2018-10-29--go-core-dev-team-weekly.md

```diff
@@ -0,0 +1,189 @@
+# 💫Golang Core Dev Team Weekly Sync 🙌🏽 October 29, 2018
+
+- **Meeting Lead:** @eingenito
+- **Notetaker:**
+- **Attendees:**
+  - bigs
+  - erikingenito
+  - hannah howard
+  - warpfork
+  - djdv
+  - Adin Schmahmann
+  - Molly Mackinlay
+  - David Dias
+  - Kevin Atkinson
+  - Hector Sanjuan
+  - Travis Person
+- **Recording:** _add link to recording once it's online_
+
+## Agenda
+
+- Ask everyone to put their name into the list of attendees
+- Round of updates
+  - What have you accomplished since the last Weekly?
+  - Were there any blockers? If so, which ones? Is it still blocked? Why?
+  - What is the next important thing you should focus on?
+- Ask for general questions. Could be things like:
+  - I'm stuck with something, I don't know who to ask. Who knows who to ask?
+  - Who can help me with xyz?
+- Plan this week
+  - Select issues to work on
+- Review remaining issues if there is time left
+
+
+## Notes
+
+### Week Update
+
+@name
+ - Done:
+ - Blocked:
+ - Next:
+
+
+@marten-seemann
+
+- Done: completed the switch to tls-tris for QUIC (still a bunch of issues to fix though)
+- Blocked: nothing
+- Next: preparing for the IETF meeting next week
```

Comment:
Anything that the group can do in order to prepare for the meeting? Does the IETF QUIC group know about libp2p? Are they excited about it?

Reply:
I'm preparing for the meeting by following the discussions of multiple WGs and reading drafts that might be interesting for us at some point in the future.
Of course, I'm bringing up p2p in general, and libp2p and IPFS from time to time.

Reply:
Might be valuable to point out that there is a draft spec for HTTP2-P2P https://github.com/ipfs/specs/issues/21